src/restarts/detail/h5writer.hpp

```diff
@@ -1,12 +1,10 @@
-
 #ifndef PHARE_DETAIL_RESTART_HIGHFIVE_HPP
 #define PHARE_DETAIL_RESTART_HIGHFIVE_HPP
 
-#include "core/logger.hpp"
-#include "core/utilities/types.hpp"
 
+#include "core/utilities/mpi_utils.hpp"
 #include "restarts/restarts_props.hpp"
-
+#include "initializer/data_provider.hpp"
```

Comment:
I don't get it

Reply:
you don't get what?

Reply:
data provider adds PhareDict

Reply:
it was implicit before I guess, cause if it was needed it wouldn't compile without this,

just seems a bit odd on it's own without other edits in this file